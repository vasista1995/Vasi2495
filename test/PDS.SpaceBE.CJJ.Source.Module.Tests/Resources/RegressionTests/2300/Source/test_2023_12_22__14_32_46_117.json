{
  "Machine": "LAW033",
  "WireSize": "125",
  "WireType": "Al",
  "Classification": "Routine check(Daily SPC)",
  "Operator": "30156566",
  "ParameterClass": "PPAP",
  "Segment": "IFPS",
  "Module": "MiniFP",
  "Package": "CIPOS MINI",
  "Device": "IKCM15F60GA",
  "FourDReport": "*",
  "SpecialCharacteristics": "ISC",
  "SpaceRawValues": [
    {
      "BreakPoint": "Rawvalueslist",
      "SampleId": 243031308,
      "Seqnr": 5,
      "Value": 368.06,
      "ExternalFlagged": "N",
      "InternalFlagged": "N",
      "NumViolations": 0
    },
    {
      "BreakPoint": "Rawvalueslist",
      "SampleId": 243031308,
      "Seqnr": 4,
      "Value": 336.97,
      "ExternalFlagged": "N",
      "InternalFlagged": "N",
      "NumViolations": 0
    },
    {
      "BreakPoint": "Rawvalueslist",
      "SampleId": 243031308,
      "Seqnr": 3,
      "Value": 339.97,
      "ExternalFlagged": "N",
      "InternalFlagged": "N",
      "NumViolations": 0
    },
    {
      "BreakPoint": "Rawvalueslist",
      "SampleId": 243031308,
      "Seqnr": 2,
      "Value": 374.81,
      "ExternalFlagged": "N",
      "InternalFlagged": "N",
      "NumViolations": 0
    },
    {
      "BreakPoint": "Rawvalueslist",
      "SampleId": 243031308,
      "Seqnr": 1,
      "Value": 335.94,
      "ExternalFlagged": "N",
      "InternalFlagged": "N",
      "NumViolations": 0
    }
  ],
  "MaterialType": "Schweizer",
  "UniqueSourceKey": "CJJ:BE:2Z3490HU:::Force_ShearOnIGBTGate:50242:243031308:0",
  "PKey": "CJJ:BE:2Z3490HU:::Force_ShearOnIGBTGate:50242:243031308:0",
  "ParameterName": "Force_ShearOnIGBTGate",
  "ParameterUnit": "g",
  "SampleDate": "2023-12-10T22:50:55Z",
  "SampleDateUtc": "2023-12-10T13:50:55Z",
  "SampleSize": 5,
  "Sigma": "18,7293392835946",
  "Mean": "351,15",
  "Median": "339,97",
  "Min": "335,94",
  "Max": "374,81",
  "Q1": "336,97",
  "Q3": "368,06",
  "LdsID": "2300",
  "IsCalcParam": 0,
  "MeanCntrlLow": 267.69565978771,
  "MeanCntrlTarget": 344.508320655397,
  "MeanCntrlHigh": 421.320981523085,
  "RangeCntrlHigh": 149.190324352651,
  "SigmaCntrlHigh": 58.4973699402583,
  "SpecLow": 150.0,
  "SpecTarget": 200.0,
  "CtrlMeanHighEnabled": "Y",
  "CtrlMeanLowEnabled": "Y",
  "CtrlMeanTargetEnabled": "Y",
  "CtrlLowEnabled": "N",
  "CtrlHighEnabled": "N",
  "CtrlRangeLowEnabled": "N",
  "CtrlRangeHighEnabled": "Y",
  "CtrlSigmaLowEnabled": "N",
  "CtrlSigmaCenterEnabled": "N",
  "CtrlRangeCenterEnabled": "N",
  "CtrlCenterEnabled": "N",
  "CtrlSigmaHighEnabled": "Y",
  "SpecLowEnabled": "Y",
  "SpecHighEnabled": "N",
  "CreatedDate": "2023-12-10T22:50:55Z",
  "CreatedDateUtc": "2023-12-10T13:50:55Z",
  "UpdatedDate": "2023-12-10T22:50:55Z",
  "UpdatedDateUtc": "2023-12-10T13:50:55Z",
  "RvStoreFlag": "Y",
  "Lot": "2Z3490HU",
  "ChannelId": "50242",
  "CkcId": "0",
  "ChannelName": "Wedge_ShearForce/MiniFP/Al/125/IGBTGate",
  "SpaceInstanceName": "BE",
  "Sitekey": "CJJ",
  "CalcCreated": "09/17/2023 15:13:45",
  "ILdsID": 0,
  "AcceptFlag": "Y",
  "UpdatedBy": " ",
  "SpecTargetOrigin": "V",
  "ViolCount": 0,
  "ExtMaMV": 343.992571428571,
  "ExtEwmaMV": 346.702310267989,
  "ExtMsMV": 0.135764501987822,
  "OutOfOrder": "N",
  "EwmaS": 17.0065582815929,
  "EwmaR": 38.9861913371803,
  "Calc2ID": 601898,
  "LimitEnable": "YYNYYNYNNYYYYYYYYYYNNNYYYNYYNNNYNNNNN",
  "ExtEwmaMVLCL": 318.888084660607,
  "ExtEwmaMVCenter": 344.497809218722,
  "ExtEwmaMVUCL": 370.107533776837,
  "ExtMaMVLCL": 325.138677125683,
  "ExtMaMVCenter": 344.497809218722,
  "ExtMaMVUCL": 363.856941311761,
  "EwmaSLCL": 4.36091236906326,
  "EwmaSCenter": 16.0615534064002,
  "EwmaSUCL": 27.7621944437371,
  "EwmaRLCL": 10.5465971880173,
  "EwmaRCenter": 39.6911746384872,
  "EwmaRUCL": 68.8357520889572,
  "ExtMSMVLCL": 0.025898865694378,
  "ExtMSMVCenter": 12.2131274934431,
  "ExtMSMVUCL": 49.0606061153418,
  "ClOrigin": 0,
  "MVLal": 0.0,
  "MVUal": 0.0,
  "RawLal": 200.0,
  "RawUal": 449.669424619215,
  "SigmaLal": -10.6226061589745,
  "SigmaUal": 126.625888633245,
  "RangeLal": -28.710481207792,
  "RangeUal": 345.193765074065,
  "ExtSpecLower": 0.0,
  "ExtSpecUpper": 0.0,
  "ExtSpecLimEnable": "NN",
  "CalculationStrategy": "ND",
  "ChannelType": "N",
  "ChState": "Online"
}