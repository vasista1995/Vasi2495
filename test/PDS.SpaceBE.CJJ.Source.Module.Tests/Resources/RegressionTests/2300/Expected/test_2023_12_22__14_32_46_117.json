{
  "IdSource": "SPACEACT2:CJJ:BE:2Z3490HU:::Force_ShearOnIGBTGate:50242:243031308:0",
  "Data_flat_Metadata": {
    "Machine": "LAW033",
    "Classification": "Routine check(Daily SPC)",
    "Operator": "30156566",
    "Package": "CIPOS MINI",
    "WireSize": "125",
    "ParameterClass": "PPAP",
    "Segment": "IFPS",
    "Module": "MiniFP",
    "Device": "IKCM15F60GA",
    "FourDReport": "*",
    "SpecialCharacteristics": "ISC",
    "WireType": "Al",
    "MaterialType": "Schweizer",
    "Lot": "2Z3490HU",
    "ParameterName": "Force_ShearOnIGBTGate",
    "ParameterUnit": "g",
    "SiteKey": "CJJ",
    "ChannelId": "50242",
    "SpaceInstanceName": "BE",
    "CreatedTimestampUtc": "2023-12-10T13:50:55Z",
    "CreatedTimestamp": "2023-12-10T22:50:55Z",
    "UpdatedTimestamp": "2023-12-10T22:50:55Z",
    "UpdatedTimestampUtc": "2023-12-10T13:50:55Z",
    "ExportedTimestampUtc": "2023-12-10T22:50:55Z",
    "ExportedTimestamp": "2023-12-10T22:50:55Z",
    "SampleTimestampUtc": "2023-12-10T13:50:55Z",
    "SampleTimestamp": "2023-12-10T22:50:55Z",
    "SourceDataLevel": "C",
    "ChannelName": "Wedge_ShearForce/MiniFP/Al/125/IGBTGate",
    "RvStoreFlag": "Y",
    "CalcCreated": "09/17/2023 15:13:45",
    "ILdsID": 0,
    "AcceptFlag": "Y",
    "UpdatedBy": " ",
    "ClOrigin": 0,
    "OutOfOrder": "N",
    "Calc2ID": 601898,
    "ViolCount": 0,
    "CalculationStrategy": "ND",
    "LdsID": "2300",
    "IsCalcParam": 0,
    "ChannelType": "N",
    "ChState": "Online"
  },
  "Data_1list_RawValues": {
    "Data_flat_Limits": {
      "CkcId": "0",
      "MeasurementSpecLimits": {
        "SpecHighEnabled": "N",
        "SpecLowEnabled": "Y",
        "SpecLow": 150.0,
        "SpecTarget": 200.0,
        "SpecTargetOrigin": "V",
        "ExtSpecLimEnable": "NN",
        "ExtSpecUpper": 0.0,
        "ExtSpecLower": 0.0,
        "LimitEnable": "YYNYYNYNNYYYYYYYYYYNNNYYYNYYNNNYNNNNN"
      },
      "SpaceAggregates": {
        "Mean": "351,15",
        "Median": "339,97",
        "Sigma": "18,7293392835946",
        "Min": "335,94",
        "Max": "374,81",
        "Q1": "336,97",
        "Q3": "368,06"
      },
      "ControlLimits": {
        "ctrl_mean_low": 267.69565978771,
        "CtrlMeanLowEnabled": "Y",
        "ctrl_mean_target": 344.508320655397,
        "CtrlMeanTargetEnabled": "Y",
        "ctrl_mean_high": 421.320981523085,
        "CtrlMeanHighEnabled": "Y",
        "CtrlLowEnabled": "N",
        "CtrlSigmaCenterEnabled": "N",
        "CtrlRangeCenterEnabled": "N",
        "CtrlCenterEnabled": "N",
        "CtrlHighEnabled": "N",
        "CtrlSigmaLowEnabled": "N",
        "ctrl_sigma_high": 58.4973699402583,
        "CtrlSigmaHighEnabled": "Y",
        "CtrlRangeLowEnabled": "N",
        "ctrl_range_high": 149.190324352651,
        "CtrlRangeHighEnabled": "Y",
        "ExtMaMV": 343.992571428571,
        "ExtEwmaMV": 346.702310267989,
        "ExtMsMV": 0.135764501987822,
        "EwmaS": 17.0065582815929,
        "EwmaR": 38.9861913371803,
        "ExtEwmaMVLCL": 318.888084660607,
        "ExtEwmaMVCenter": 344.497809218722,
        "ExtEwmaMVUCL": 370.107533776837,
        "ExtMaMVLCL": 325.138677125683,
        "ExtMaMVCenter": 344.497809218722,
        "ExtMaMVUCL": 363.856941311761,
        "EwmaSLCL": 4.36091236906326,
        "EwmaSCenter": 16.0615534064002,
        "EwmaSUCL": 27.7621944437371,
        "EwmaRLCL": 10.5465971880173,
        "EwmaRCenter": 39.6911746384872,
        "EwmaRUCL": 68.8357520889572,
        "ExtMSMVLCL": 0.025898865694378,
        "ExtMSMVCenter": 12.2131274934431,
        "ExtMSMVUCL": 49.0606061153418,
        "MVLal": 0.0,
        "MVUal": 0.0,
        "RawLal": 200.0,
        "RawUal": 449.669424619215,
        "SigmaLal": -10.6226061589745,
        "SigmaUal": 126.625888633245,
        "RangeLal": -28.710481207792,
        "RangeUal": 345.193765074065
      }
    },
    "Data_1list_RawValues": [
      {
        "SampleId": 243031308,
        "Seqnr": 5,
        "Value": 368.06,
        "InternalFlagged": "N",
        "ExternalFlagged": "N",
        "NumViolations": 0
      },
      {
        "SampleId": 243031308,
        "Seqnr": 4,
        "Value": 336.97,
        "InternalFlagged": "N",
        "ExternalFlagged": "N",
        "NumViolations": 0
      },
      {
        "SampleId": 243031308,
        "Seqnr": 3,
        "Value": 339.97,
        "InternalFlagged": "N",
        "ExternalFlagged": "N",
        "NumViolations": 0
      },
      {
        "SampleId": 243031308,
        "Seqnr": 2,
        "Value": 374.81,
        "InternalFlagged": "N",
        "ExternalFlagged": "N",
        "NumViolations": 0
      },
      {
        "SampleId": 243031308,
        "Seqnr": 1,
        "Value": 335.94,
        "InternalFlagged": "N",
        "ExternalFlagged": "N",
        "NumViolations": 0
      }
    ],
    "SampleSize": 5
  },
  "ProductionAction": {
    "Id": "SPACEACT1:CJJ:BE:::Force_ShearOnIGBTGate:50242",
    "Type": "SpaceAction"
  },
  "Item": {
    "Id": "Measlot:2Z3490HU",
    "IdSystemName": "Measlot",
    "Type": "Lot"
  },
  "SystemLog": {
    "ExtractedTimestampUtc": "0001-01-01T00:00:00",
    "EnqueuedTimestampUtc": "0001-01-01T00:00:00",
    "UniqueSourceKey": "CJJ:BE:2Z3490HU:::Force_ShearOnIGBTGate:50242:243031308:0"
  }
}